packages/ui/src/components/atoms/Avatar/index.tsx

```diff
@@ -7,43 +7,44 @@ export type TAvatar = HTMLAttributes<HTMLDivElement> & {
   src?: string
   alt: string
   avatarCss?: Interpolation<ThemedStyledProps<TAvatar, unknown>>
-  mediaQuery?: { [key: string]: string; }
+  mediaQuery?: { [key: string]: string }
   forwardedRef?: React.RefObject<HTMLImageElement>
   onClick?: () => void
 }
 
 const StyledAvatar = styled.img<TAvatar>`
- ${props => props.avatarCss};
+  ${(props) => props.avatarCss};
   object-fit: cover;
   appearance: auto;
   text-align: center;
   cursor: pointer;
+
   @media only ${device.Tablet} {
-    ${(props) => props.mediaQuery}
+    ${(props) => props.mediaQuery};
   }
 `
+
 const defaultMediaQuery = {
   width: '118px',
   height: '118px',
   border: 'none',
 }
+
 const defaultAvatarCss = {
   width: '97px',
   height: '97px',
   borderRadius: '50%',
   backgroundColor: colors.gray.gray5,
   border: `1px solid ${colors.gray.gray4}`,
-
 }
+
 export const Avatar: FC<TAvatar> = ({
   src = defaultAvatar,
   alt,
-  avatarCss = {
-    defaultAvatarCss
-  },
+  avatarCss = { ...defaultAvatarCss },
```

Comment:
Is there a reason for this change? Sorry, just trying to understand!

Reply:
Without the spread operator wouldn't pass the styles for mobile in Storybook, that's why. Also, Prettier passed it to the same line.

Reply:
I didn't realize the default styles were not working, thanks!